src/rdfTypes/GeometryInfo.cpp

```diff
@@ -57,7 +60,15 @@ std::optional<GeometryInfo> GeometryInfo::fromWktLiteral(std::string_view wkt) {
     return std::nullopt;
   }
 
-  return GeometryInfo{type, boundingBox.value(), centroid.value()};
+  double area = std::numeric_limits<double>::quiet_NaN();
+  try {
+    area = computeMetricArea(parsed.value());
+  } catch (const InvalidPolygonError&) {
+    AD_LOG_WARN << "Could not compute area of WKT literal `" << wkt << "`."
+                << std::endl;
+  }
+
+  return GeometryInfo{type, boundingBox.value(), centroid.value(), {area}};
```

Comment:
We should discuss the pro and cons of `NaN` vs. `nullopt` for errors etc. (what is the area of a point/line, is it zero or the error value?